### DISCUSSION TITLE
Is it possible to see a complete history of pull requests for a repository?

### DISCUSSION THREAD

**Discussion by jacquegothard** [community/community#21776] [New to GitHub]
<p>Hi! I’d like to look through some past pull requests in my team’s repo: is there a cached PR history somewhere?</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey <a class="mention" href="/jacquegothard">@jacquegothard</a>, Welcome to the Community Forum! The best way to do this would be via the GitHub API found here: <a href="https://developer.github.com/v3/pulls/" rel="nofollow noopener">https://developer.github.com/v3/pulls/</a></p>
